api/clients/v2/payloaddispersal/payload_disperser.go

```diff
@@ -75,12 +76,11 @@ func (pd *PayloadDisperser) SendPayload(
 ) (*coretypes.EigenDACert, error) {
 
 	probe := pd.stageTimer.NewSequence()
```

Comment:
why did you remove the `defer probe.End()`?

Reply:
This was done in error, I've added it back in. We would have noticed this if we had deployed this code... the result would be that the graph would show a runaway number of the last stage.